src/components/molecules/ourstoryCard/OurStoryCard.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styles from "./OurStoryCard.module.scss";
+import Image from "next/image";
+
+interface OurStoryCardProps {
+  image: string;
+  story: string;
```

Comment:
It would be nice if the story type was string and ReactNode. I will allow us to style our story text in the future.